Sage/carbon
# fix(text-editor): update typo to ensure focused toolbar button has relative position

### Proposed behaviour


Fix typo in styling to ensure the button is relative positioned to prevent the next button overlaying the focus outlines
![image](https://github.com/Sage/carbon/assets/44157880/d303b3f0-042f-4987-9532-175d750b28bf)


### Current behaviour



![image](https://github.com/Sage/carbon/assets/44157880/1c3fe281-cca2-4822-9777-c76f50cb31e1)

### Checklist




#### QA


### Additional context



### Testing instructions



`text-editor--default` story can be used:
- ensure `focusRedesignOptOut` flag is **NOT** set
- click `italic` button to make it active and then focus the `bold` one and ensure the focus outline overlays the next button correctly

The following CodeSandbox is an example of the broken behaviour.
You can see the new behaviour by looking at the version in the comment by `codesandbox[bot]`.
